url8e6440c8aa_robdimsdale_project_euler_tgz-pJ8-com_rmd_personal_projecteuler_MainJ8/src/com/rmd/personal/projecteuler/P049.java

```diff
@@ -0,0 +1,106 @@
+package com.rmd.personal.projecteuler;
+
+public class P049 implements Problem {
+
+    private static final int DEFAULT_END = 9999;
+    private static final int DEFAULT_START = 1488;
+
+    private int start;
+    private int end;
+
+    public P049() {
+        this.setStart(DEFAULT_START);
+        this.setEnd(DEFAULT_END);
+    }
+
+    public int getEnd() {
+        return end;
+    }
+
+    public void setEnd(int end) {
+        this.end = end;
+    }
+
+    public int getStart() {
+        return start;
+    }
+
+    public void setStart(int start) {
+        this.start = start;
+    }
+
+    @Override
+    public String getDescription() {
+        return "The arithmetic sequence, 1487, 4817, 8147, in which each of the terms increases by 3330, "
+                + "is unusual in two ways: (i) each of the three terms are prime, and, "
+                + "(ii) each of the 4-digit numbers are permutations of one another.\n\n"
+                + "There are no arithmetic sequences made up of three 1-, 2-, or 3-digit primes, "
+                + "exhibiting this property, but there is one other 4-digit increasing sequence.\n\n"
+                + "What 12-digit number do you form by concatenating the three terms in this sequence?";
+    }
+
+    @Override
+    public String run() {
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The cyclomatic complexity of this method is 16. By comparison, 99% of the methods in the CodeGuru reference dataset have a lower cyclomatic complexity. This indicates the method has a high number of decisions and it can make the logic difficult to understand and test. We recommend that you simplify this method or break it into multiple methods.